openshift/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM postgres:10
```

Comment:
It looks like there is no difference between this file in the `kubernetes` directory and here.
Can we just make a symlink?

Reply:
This file is modified at the bottom to set the permissions appropriately for OpenShift assigned UID/GID to work. 

The updated content is: 
```
# Set directory permissions for OpenShift
RUN chmod a+w /etc/passwd \
    && chgrp -R 0 /home/postgres \
    && chmod -R g+rwX /home/postgres
```